src/libraries/skinDynLib/include/iCub/skinDynLib/common.h

```diff
@@ -72,15 +72,7 @@ const std::string SkinPart_s[] = {
 };
 
 // *** Mapping from SKIN PARTs to BODY PARTs ***
-// To represent this information you could use either a map<SkinPart,BodyPart>
-// or a vector of struct{SkinPart; BodyPart;}. I chose the second, but I
-// left the implementation for the first option (actually two different implementations)
-#define OPTION2 // this variable allows to switch between the different implementations
-#ifdef OPTION1
-std::map<SkinPart,BodyPart> createSkinPart_2_BodyPart();
-const std::map<SkinPart,BodyPart> SkinPart_2_BodyPart = createSkinPart_2_BodyPart();
-#endif
-#ifdef OPTION2
+// To represent this information we use a vector of struct{SkinPart; BodyPart;}
```

Comment:
For now it was just a cleanup of the other versions that were not compiling anymore. I still think that those kind of mappings should not be handled in a compile-time fixed structure, but this PR is not changing this.